docs/source-pytorch/extensions/callbacks_state.rst

```diff
@@ -26,6 +26,9 @@ the following example.
             # note: we do not include `verbose` here on purpose
             return self._generate_state_key(what=self.what)
 
+        def _generate_state_key(self, **kwargs: Any) -> str:
+            return f"{self.__class__.__qualname__}{repr(kwargs)}"
```

Comment:
```suggestion
            return f"Counter[what={self.what}]"
```
I think this is more readable for the example

Reply:
What about this?

```
@property
def state_key(self):
      return f"Counter[what={self.what}]"
```

Reply:
Yeah that's what I meant 😃 I would avoid the _generate_state_key method completely.